src/Microsoft.AspNet.SignalR.Redis/RedisScaleoutOptions.cs

```diff
@@ -23,12 +23,12 @@ public RedisScaleoutOptions(string connectionString, string eventKey)
         {
             if (String.IsNullOrEmpty(connectionString))
             {
-                throw new ArgumentNullException("connectionString");
+                throw new ArgumentNullException(nameof(connectionString));
             }
 
             if (String.IsNullOrEmpty(eventKey))
             {
-                throw new ArgumentNullException("eventKey");
+                throw new ArgumentNullException(nameof(eventKey));
```

Comment:
@hishamco don't worry about further changes here.  We apparently have a larger problem with this misleading `Exception` type...